src/main/scala/org/globalforestwatch/layers/Plantations.scala

```diff
@@ -21,3 +21,9 @@ case class Plantations(gridTile: GridTile) extends StringLayer with OptionalILay
     case _ => ""
   }
 }
+
+case class PlantationsBool(gridTile: GridTile) extends BooleanLayer with OptionalILayer {
```

Comment:
minor, but it might be more consistent to rename the class above to PlantationType, and call this one IsPlantation